.eslintrc

```diff
@@ -0,0 +1,18 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true,
+    "node": true
+  },
+  "extends": "eslint:recommended",
```

Comment:
For a next task, take a look at other predefined configs, also from other companies (e.g. airbnb's config) and see if you like them more :smile: 

Personally, I always use their config as a base, but analyze multiple ones and see which ones you like, which ones you don't (preffereably in a spreadsheet/document) :slightly_smiling_face:

Reply:
That's excellent suggestion i will investigate that.